.bemrc

```diff
@@ -1,22 +1,27 @@
-{
-    "root" : true,
-    "levels" : {
-        "./blocks" : {
-            "scheme" : "nested",
-            "naming" : "react",
-            "default" : true
+module.exports = {
```

Comment:
I think we don't need to use modules if it's not js. Nobody can't require this file.